packages/client/wallets/smart-wallet/src/blockchain/wallets/EVMSmartWallet.ts

```diff
@@ -112,4 +125,68 @@ export class EVMSmartWallet {
     public async nfts() {
         return this.crossmintService.fetchNFTs(this.address, this.chain);
     }
+
+    /**
+     * Sends a contract call transaction and returns the hash of a confirmed transaction.
+     * @param address the address of the contract to be called
+     * @param abi the ABI of the contract - ***should be defined as a typed variable*** to enable type checking of the contract arguments, see https://viem.sh/docs/typescript#type-inference for guidance
+     * @param functionName the name of the smart contract function to be called
+     * @param args the arguments to be passed to the function
+     * @param resendCallback a callback function that will be called if the transaction needs to be resent. Returns a value that controls transaction rebroadcasting See the type of `TransactionServiceResendCallback` for more information.
+     * @returns The transaction hash.
+     * @throws `SendTransactionError` if the transaction fails to send. Contains the error thrown by the viem client.
+     * @throws `SendTransactionExecutionRevertedError`, a subclass of `SendTransactionError` if the transaction fails due to a contract execution error.
+     *
+     * **Passing a typed ABI:**
+     * @example
+     * const abi = [{
+     *   "inputs": [
+     *     {
+     *       "internalType": "address",
+     *         "name": "recipient",
+     *         "type": "address"
+     *       },
+     *   ],
+     *   "name": "mintNFT",
+     *   "outputs": [],
+     *   "stateMutability": "nonpayable",
+     *   "type": "function"
+     * }] as const;
+     *
+     * await wallet.sendTransaction({
+     *   address: contractAddress,
+     *   abi,
+     *   functionName: "mintNFT",
+     *   args: [recipientAddress],
+     * });
+     */
+    public async sendTransaction<
```

Comment:
afaik this doesn't support sending eth to another address, it should right?

Maybe something like:

```TS
account.sendTransaction({
            contractWrite: {
                abi: CollectionABI,
                functionName: "mintToCustomErrorTester",
                args: [account.address, BigInt(98)],
            }
            address: "0x5c030a01e9d2c4bb78212d06f88b7724b494b755",
            value: BigInt(5),
});
```

With :
- `contractWrite` as an optional property of the function parameters, AND
- all properties of `contractWrite` as non optional.

That way clients/dev must always include the ABI when interacting with a contract